internal/app/tfsec/config/config.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/tfsec/tfsec/internal/app/tfsec/debug"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type Config struct {
+	SeverityOverrides map[string]string `json:"severity_overrides,omitempty" yaml:"severity_overrides,omitempty"`
+}
+
+func LoadConfig(configFilePath string) (*Config, error) {
+	var config = &Config{}
+
+	if _, err := os.Stat(configFilePath); err != nil {
+		if os.IsNotExist(err) {
+			debug.Log("Failed to load the config file, not found. %s", configFilePath)
+			return config, nil
+		}
```

Comment:
If we hit some other error we should return it:
```suggestion
		} else {
		    return nil, err
		}
```